### PR TITLE
inline image push notifications

### PR DESCRIPTION
**Pull request by radixdev**
# Pull Request/Issue Resolution

**Description of Change:**
Documentation for `Android Inline Image Push`

### Is this change associated with a Braze feature/product release?
- [x] Yes (Android SDK is already released. Dashboard previewer/composer in ~3 weeks max)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -1 +1 @@
-2.5.1
+2.6.5
```
Comment (radixdev) [inline .ruby-version:1]: this repo wouldn't build unless i bumped the version

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/push_story.md [hunk 2]
```diff
@@ -1,12 +1,14 @@
 ---
-nav_title: Push Story
+nav_title: Push Stories
```
Comment (radixdev) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/push_story.md:2]: Moved to be under "push notifications"

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+platform: Android
+page_order: 14
+
+---
+
+# Android Inline Image Push
+
+![Android Inline Image Push example]({% image_buster /assets/img/android/push/inline_image_push_android_1.png %}){: width="55%"}
+
+Inline Image push notifications are custom push notifications by Braze that show images inline without the need to manually expand like standard push notifications. Although they use custom rendering, they adhere to the Material Notification guidelines by Android as much as possible in order to resemble regular Android OS push notifications.
```
Comment (davidbielik) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:12]: want to add a link to those standards?

Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:12]: No. It's a source code standard really, not one published anywhere

**Review by davidbielik — DISMISSED** [review]
thanks!

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md (hunk 3 above)
Comment (Bucimis) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:12]: "Although they use custom rendering, they adhere to the Material Notification guidelines by Android as much as possible in order to resemble regular Android OS push notifications." -> "Although they use custom rendering, they were built in adherence to Material Notification guidelines by Android" imo, we don't need to draw excessive attention to the fact that they're custom rendered

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+
+# Usage Overview
+
+No extra integration is required for Android Inline Image Push after upgrading to Braze Android SDK 10.0.0. Inline Image push notifications require Android M+ devices. Devices lower than Android M will see standard big image push notifications instead of inline images. Additionally, push action buttons are not supported when using this notification type.
+
+# Inline Image Aspect Ratio
+
+Unlike regular Android push notifications, Inline Image push images are in a 3:2 aspect ratio.
```
Comment (Bucimis) [inline _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:20]: What happens to devices<M? I guess the cropping may be slightly weird?

**Review by Bucimis — APPROVED** [review]

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md (hunk 3 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:12]: I just removed the whole section actually.

File: _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md (hunk 4 above)
Comment (radixdev) [reply _docs/_developer_guide/platform_integration_guides/android/push_notifications/inline_image_push.md:20]: Yup, they'll appear like a regular push. The OS will scale it to fit and do no cropping. I just did a side-by-side of a 2:1 and 3:2 regular push and can't really tell the difference.

**Review by KellieHawks — APPROVED** [review]
lgtm

**Pull request merged**